packages/playground/_stories/main/Panel/Panel.stories.ts

```diff
@@ -23,77 +28,72 @@ export default {
-<h3>Panel with List</h3>
-	<div class="snippet">
-		<ui5-panel accessible-role="Complementary" header-text="Select your country" class="full-width">
-			<ui5-list id="myList1" mode="MultiSelect">
-				<ui5-li key="country1">Argentina</ui5-li>
-				<ui5-li key="country2">Bulgaria</ui5-li>
-				<ui5-li key="country3">China</ui5-li>
-				<ui5-li key="country4">Germany</ui5-li>
-				<ui5-li key="country5">Hungary</ui5-li>
-				<ui5-li key="country6">England</ui5-li>
-				<ui5-li key="country7">USA</ui5-li>
-				<ui5-li key="country8">Canada</ui5-li>
-			</ui5-list>
-		</ui5-panel>
-	</div>
-`;
+export const PanelWithList = Template.bind({});
+PanelWithList.args = {
+	default: '<ui5-list mode="MultiSelect"><ui5-li key="country1">Argentina</ui5-li><ui5-li key="country2">Bulgaria</ui5-li><ui5-li key="country3">China</ui5-li><ui5-li key="country4">Germany</ui5-li><ui5-li key="country5">Hungary</ui5-li><ui5-li key="country6">England</ui5-li><ui5-li key="country7">USA</ui5-li><ui5-li key="country8">Canada</ui5-li></ui5-list>',
+	headerText: "Select your country",
+	accessibleRole: PanelAccessibleRole.Complementary
+};
+PanelWithList.storyName = "Panel with List";
 
 
-export const Template2: StoryFn = () => html`
-<h3>Fixed Panel (Can't be Collapsed/Expanded)</h3>
-	<div class="snippet panel-snippet">
-		<ui5-panel class="full-width" fixed="" accessible-role="Complementary" header-text="Country Of Birth">
-			<ui5-list id="myList2" mode="SingleSelectBegin">
-				<ui5-li key="country1">Argentina</ui5-li>
-				<ui5-li key="country2">Bulgaria</ui5-li>
-				<ui5-li key="country3">China</ui5-li>
-				<ui5-li key="country4">Germany</ui5-li>
-			</ui5-list>
-		</ui5-panel>
-	</div>
-`;
+export const FixedPanel = Template.bind({});
+FixedPanel.args = {
+	default: '<ui5-list mode="SingleSelectBegin" ><ui5-li key="country1">Argentina</ui5-li><ui5-li key="country2">Bulgaria</ui5-li><ui5-li key="country3">China</ui5-li><ui5-li key="country4">Germany</ui5-li></ui5-list>',
+	headerText: "Country Of Birth",
+	accessibleRole: PanelAccessibleRole.Complementary,
+	fixed: true
+};
+FixedPanel.storyName = "Fixed Panel (Can't be Collapsed/Expanded)";
 
 
-export const Template3: StoryFn = () => html`
-<h3>Panel with Custom Header</h3>
-	<div class="snippet">
-		<ui5-panel accessible-role="Complementary" class="full-width">
-			<!-- Panel header -->
-			<div slot="header" class="header">
-				<ui5-title>Countries</ui5-title>
-				<div>
-					<ui5-button>Edit</ui5-button>
-					<ui5-button design="Emphasized">Add</ui5-button>
-					<ui5-button design="Negative">Remove</ui5-button>
-				</div>
-			</div>
-			<ui5-list id="myList1" mode="MultiSelect">
-				<ui5-li key="country1">Argentina</ui5-li>
-				<ui5-li key="country2">Bulgaria</ui5-li>
-				<ui5-li key="country3">China</ui5-li>
-			</ui5-list>
-		</ui5-panel>
-	</div>
-`;
+export const PanelCustomHeader = Template.bind({});
+PanelCustomHeader.decorators = [
+	(story) => {
+		return html`
+		<style>
```

Comment:
<img width="1001" alt="Screenshot 2023-04-20 at 9 55 33" src="https://user-images.githubusercontent.com/11913394/233286301-39ff5acc-8ddd-4cc3-acf2-2f7a36da3a5e.png">
This is not formatted correctly.

Reply:
Done